src/tyr/route_serializer_osrm.cc

```diff
@@ -33,6 +35,86 @@ struct NamedSegment {
   float distance;
 };
 
+const constexpr double TILE_SIZE = 256.0;
+static constexpr unsigned MAX_ZOOM = 18;
+static constexpr unsigned MIN_ZOOM = 1;
+// this is an upper bound to current display sizes
+static constexpr double VIEWPORT_WIDTH = 8 * TILE_SIZE;
+static constexpr double VIEWPORT_HEIGHT = 5 * TILE_SIZE;
+static double INV_LOG_2 = 1. / std::log(2);
+const constexpr double DEGREE_TO_RAD = 0.017453292519943295769236907684886;
+const constexpr double RAD_TO_DEGREE = 1. / DEGREE_TO_RAD;
```

Comment:
a couple of these we have already in `midgard/constants.h`

Reply:
no these are mercator defaults